src/main/java/org/openstreetmap/atlas/generator/world/WorldAtlasGenerator.java

```diff
@@ -59,6 +64,20 @@
                     + " always be attempted regardless of the number of countries it intersects according to the"
                     + " country boundary map's grid index.",
             StringConverter.IDENTITY, Optionality.OPTIONAL);
+    public static final Switch<File> PBF_WAY_CONFIGURATION = new Switch<>("osmPbfWayConfiguration",
+            "The path to the configuration file that defines which PBF Ways becomes an Atlas Entity.",
+            File::new, Optionality.OPTIONAL);
+    public static final Switch<File> PBF_NODE_CONFIGURATION = new Switch<>(
+            "osmPbfNodeConfiguration",
+            "The path to the configuration file that defines which PBF Nodes becomes an Atlas Entity.",
+            File::new, Optionality.OPTIONAL);
+    public static final Switch<File> PBF_RELATION_CONFIGURATION = new Switch<>(
+            "osmPbfRelationConfiguration",
+            "The path to the configuration file that defines which PBF Relations becomes an Atlas Entity",
+            File::new, Optionality.OPTIONAL);
+    // filter that does no filtering
+    private static final ConfiguredTaggableFilter PBF_NO_FILTER_CONFIGURATION = new ConfiguredTaggableFilter(
+            new StandardConfiguration(new StringResource("{\n    \"filters\": [\n\n    ]\n}")));
```

Comment:
Here the `\n`s are not necessary! `"{\"filters\": []}"` should work I think.

Reply:
I'm out of the loop here. Is this JSON?